test/test_torch.py

```diff
@@ -13436,7 +13432,7 @@ def test_addcdiv(self, device):
         def _test_addcdiv(a, alpha, b, c):
             actual = torch.addcdiv(a, b, c, value=alpha)
             # implementation of addcdiv downcasts alpha. arithmetic ops don't.
-            if not actual.dtype.is_floating_point:
+            if not (actual.dtype.is_floating_point or actual.dtype.is_complex):
```

Comment:
I don't see a test for if `alpha` is complex for addcdiv or addcmul, that would be good to add for completeness.

Reply:
ha I didn't notice that thanks! updated the tests